chain/consensus/hierarchical/subnet/manager/subnet.go

```diff
@@ -167,6 +167,8 @@ func (sh *Subnet) Close(ctx context.Context) error {
 	sh.syncer.Stop()
 	// Close message pool
 	err5 := sh.mpool.Close()
+	// Close resolver.
+	err6 := sh.r.Close()
```

Comment:
I believe err1, err2, ... are not idiomatic names for errors. Let's add TODO?

Reply:
Good point, adding TODO.